packages/mds-audit/api.ts

```diff
@@ -441,7 +441,6 @@ function api(app: express.Express): express.Express {
 
           // Validate input params
           if (
-            isValidAuditEventId(audit_event_id) &&
```

Comment:
I wonder if this was meant to validate the audit trip id?

Reply:
I'm not entirely sue what `audit_event_id` is used for (haven't dug into the audit code much besides `/vehicles`), but this line is simply a dupe of the line right below it.

Reply:
`audit_event_id` is just a unique uuid assigned to the audit event. Probably not used for much other than tracking in the db (from back before our tables had id columns). I understand it's a dupe. My point was, that perhaps it was a copy-pasta error and should have been `isValidAuditTripId(audit_trip_id)` to validate the uuid in the route param, but looking at the code it appears that check already happens in some earlier middleware.